test/unit-test/FreeRTOS_DNS_Networking/FreeRTOS_DNS_Networking_utest.c

```diff
@@ -132,6 +132,40 @@ void test_CreateSocket_success( void )
     TEST_ASSERT_EQUAL( ( Socket_t ) 235, s );
 }
 
+/**
+ * @brief Ensures that when the socket could not be created or could not be found, null is returned
+ */
+void test_BindSocket_fail( void )
+{
+    struct freertos_sockaddr xAddress;
+    Socket_t xSocket;
+    uint16_t usPort;
+    uint32_t ret;
+
+    FreeRTOS_bind_ExpectAnyArgsAndReturn( pdFALSE );
+
+    ret = DNS_BindSocket( xSocket, usPort );
```

Comment:
Here is the definition of FreeRTOS_bind return value:
```
 * @return The return value is 0 if the bind is successful.
 *         If some error occurred, then a negative value is returned.
```

So pdFALSE/pdTRUE is not the return value of FreeRTOS_bind. Neither do DNS_BindSocket.

Reply:
Updated.